buildifier/config/config_test.go

```diff
@@ -260,9 +267,16 @@ func TestValidate(t *testing.T) {
 			"native-android",
 			"native-build",
 			"native-cc",
+			"native-cc-proto",
 			"native-java",
+			native-java-lite-proto",
```

Comment:
Missing a quote here.

Reply:
```
gentestmain: ParseFile("buildifier/config/config_test.go"): buildifier/config/config_test.go:272:26: string literal not terminated (and 10 more errors)
```

Reply:
Ahhh thanks!! I was looking at protobuf warning and didn't see the error :)